src/core/createSaleorClient.ts

```diff
@@ -20,11 +22,19 @@ export const createSaleorClient = ({
   const authSDK = auth(coreInternals);
   const userSDK = user(coreInternals);
 
+  if (autologin) {
+    const token = localStorage.getItem(SALEOR_AUTH_TOKEN);
+
+    if (token) {
+      authSDK.verifyToken(token);
```

Comment:
What happens when a token is invalid?

Reply:
`data.tokenVerify.user` is null and cache is not updated

Reply:
I wonder if we should await this query? Wonder if letting it run in async would cause unnecessary page reloads. @jwm0 do you think that would be an issue?